setup.py

```diff
@@ -221,9 +221,6 @@
     # For example, the following would provide a command called `sample` which
     # executes the function `main` from this package when invoked:
     entry_points={
-        'console_scripts': [
-            'fix_links = pyworkflow.project.scripts.fix_links:main',
-        ],
         'pyworkflow.plugin': 'pyworkflowtests = pyworkflowtests'
```

Comment:
Good point!

Reply:
Maybe we can add other entry points?

Reply:
none of the project scripts have main() method. Of course, they all can be refactored to simplify the commands for users